### PR TITLE
fix nav toggle problem on ios safari

### PR DESCRIPTION
**Pull request by kgurnani96**
## Description
iOS safari executes $body event listener, when $navToggle is clicked, so i just added a condition to check if its the toggle button, to just exit the function

## Motivation and Context
fixes issue #1920

## How Has This Been Tested?
Toggled nav in chrome for mac and it worked, and tried the same in ios safari, and it worked too (yayy!)

## Types of changes
- [x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by MonkeyLeeT** [comment]
Quick fix, easy testing. +1

File: static/js/app.js [hunk 1]
```diff
@@ -64,8 +68,9 @@
 
 	// Event: Hide on ESC.
 	window.addEventListener('keydown', function(event) {
-		if (event.keyCode == 27)
+		if (event.keyCode == 27) {
 			$nav.classList.remove('visible');
+		}
```
Comment (MonkeyLeeT) [inline static/js/app.js]: Can we leave it as it is?

Comment (kgurnani96) [inline static/js/app.js]: yes, forgot to remove it as i was logging to console (oops)

Comment (kgurnani96) [inline static/js/app.js]: removed the bracket

**Pull request merged**